app/src/main/java/be/brunoparmentier/openbikesharing/app/activities/StationActivity.java

```diff
@@ -93,7 +106,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 map.setTileSource(TileSourceFactory.MAPNIK);
                 break;
             case MAP_LAYER_CYCLEMAP:
-                map.setTileSource(TileSourceFactory.CYCLEMAP);
+                map.setTileSource(new ThunderforestTileSource(context, ThunderforestTileSource.CYCLE));
```

Comment:
Doesn't this source require an API key?

Reply:
Indeed. I just made this change because the old code was not compiling, but it would be better to use a legit source instead.

Reply:
I should have remove it. I can re-push with the removing unless you think about another source to add...

Reply:
Finally re-pushed with another source (HikeBikeMap). Looking for your review. Regards.